src/Link/index.js

```diff
@@ -82,6 +91,10 @@ export default class Link extends React.PureComponent {
       return this.drawStraightPath(linkData, orientation);
     }
 
+    if (pathFunc === 'stair') {
```

Comment:
Minor (grammar): When referring to the concept of stairs or a stair-like object in English, it's usually clearer to refer to the plural form (i.e. "stairs") if you want people to think of a stair-shaped graph. Kind of counter-intuitive I know.

Reply:
I would leave `drawStairPath` untouched though. I'm only referring to the enum value for `pathFunc` :)

Reply:
I renamed it to "Step". It's more obvious i think.